apps/L1E/ticket-booking/ticket-booking-frontend/specs/components/TimePicker.test.tsx

```
@@ -0,0 +1,127 @@
+                onChange(time);
+              }
+            } catch (error) {
+              // Handle invalid time
+            }
+          } else {
+            onChange(null);
+          }
+        }}
+      />
+    );
+  };
```

ESLint

[error] Function component is not an arrow function

(react/function-component-definition)